protocol/avdl/secret_ui.avdl

```diff
@@ -37,18 +37,23 @@ protocol secretUi {
 
   string getPaperKeyPassphrase(int sessionID, string username);
 
-  record SecretStorageFeature {
+  record Feature {
     boolean allow;
+    boolean defaultValue;
+    boolean readonly;  /* if true we don't send the value back */
     string label;
   }
 
   record GUIEntryFeatures {
-    SecretStorageFeature secretStorage;
+    Feature storeSecret;
+    Feature showTyping;
   }
 
   record GUIEntryArg {
     string windowTitle;
     string prompt;
+    string submitLabel;
+    string cancelLabel;
```

Comment:
per a convo w/ @MarcoPolo and @malgorithms we can have different submit buttons if we're like logging in or signing up or entering a paper key etc